app/assets/stylesheets/modules/master_overlay.css.scss

```diff
@@ -333,21 +354,48 @@
 }
 
 .master-overlay-global-actions {
-  padding: 0;
-  border: 1px solid transparent;
-  border-radius: 3px;
+  display: inline-block;
+  vertical-align: top;
+  width: 23%;
+
+  .add-to-project, .remove-from-project {
+    font-size: 0.8em;
+    margin-top: 3px;
+    float: right;
+  }
+
+  .remove-from-project {
+    color: #c0392b;
+  }
+
+  .global-action-link {
+    font-size: 0.8em;
+  }
 
   .button {
     @include list-panel-button;
-    padding: 6px 6px;
+    margin-top: 8px;
+    display: inline-block;
+    width: 100%;
+    vertical-align: middle;
+    padding: 8px 10px;
+
+    .pill {
+      background: rgba(0,0,0,0.4);
+      border-radius: 15px;
+      text-shadow: none;
+      font-size: 0.8em;
+      padding: 2px 10px;
+      margin-left: 5px;
+      vertical-align: bottom;
+      display: inline-block;
+    }
   }
 
   .retrieve {
     background: #27ae60;
     border: none;
     color: #fff;
-    padding-left: 10px;
-    padding-right: 10px;
 
     &:hover {
       background: #219853;
```

Comment:
I realize you didn't touch this, but other buttons throughout the app have a transition between normal and hover states. Pretty sure we have a mixin for it somewhere. Would be nice to throw that on here.